js/scripts.js

```diff
@@ -1 +1,22 @@
-﻿
\ No newline at end of file
+﻿var a;
+var h;
+
+function getTriangleArea(a, h){
+	var triangleArea;
+	if ((a < 0) || (h < 0)) {
+		alert('Nieprawidłowe wartości!');
+	}
+	else {
+		triangleArea = a * h / 2;
+		return triangleArea;
+	}
+}
+console.log(getTriangleArea(10, 6));
+
+var triangle1Area = getTriangleArea(15,8);
+var triangle2Area = getTriangleArea(8,12);
+var triangle3Area = getTriangleArea(-2, 5);
+
+console.log(triangle1Area);
```

Comment:
console logujesz wartości zmiennej, a te kilka linijek wyżej wyliczasz na podstawie funkcji. Funkcja natomiast 'wypełnia' te zmienne na podstawie tego co zwraca `return`

w przypadku poprawnego wyniku zwracasz ten wynik i jest elegancko, ale w przypadku nieprawidłowych wartości nie zwracasz nic (tylko pokazujesz alert). Jak funkcja nic nie zwraca to z automatu zwraca `undefined`. Dlatego jest niekonsekwentnie bo przy dobrych liczbach widzisz w konsoli wynik, a przy złych - `undefined`.

Przerób to zadanie tak żeby przy złej wartości zamiast wyniku pojawiał się w konsoli napis 'nieprawidłowe wartości` - wyrzuć natomiast ten alert całkowicie.